### PR TITLE
Convert node frontend service to OTel 

### LINKED ISSUE #140: Convert node frontend service to OTel
Currently, only the year-service in the Node example is using OTel, every other service is using the beeline. Convert frontend service to use OTel as well, so we can demonstrate a more complex interop:
frontend (OTel) <> name-service (beeline) <> year-service (OTel)

The resulting traces should look about the same as today.

### PR DESCRIPTION
**Pull request by emilyashley**
## Which problem is this PR solving?

- closes #140 
- learned a lot about Node OTel ✨ 

## Short description of the changes
- remove beeline from frontend service
- add node auto instrumentation to frontend service, created tracing.js 
- add a beeline trace context in the name service (and message service) instead of as part of a promise in frontend service
- use `npm start` script to include tracing.js and main.js in docker (match tilt)

### CODE REVIEW

**Comment by emilyashley** [comment]
What a journey!

File: node/frontend/main.js [hunk 1]
```diff
@@ -38,18 +25,26 @@ const corsOptions = {
...
+  greetingSpan.end()
+
+  const nameSpan = tracer.startSpan('✨ call /name ✨');
   const name = await getName(nameUrl);
-  beeline.finishSpan(nameSpan);
-  const messageSpan = beeline.startSpan({ name: '✨ call /message ✨' });
+  nameSpan.setAttribute("app.user_name", name);
+  nameSpan.end()
+
+  const messageSpan = tracer.startSpan('✨ call /message ✨');
```
Comment (pkanal) [inline node/frontend/main.js:41]: Do we want to record the message we get here as an attribute as well?

**Review by pkanal — APPROVED** [review]
Looks great! ✨  Left a non-blocking comment about setting some extra attributes.

File: node/frontend/main.js (hunk 1 above)
Comment (emilyashley) [reply node/frontend/main.js:41]: ✅

**Pull request merged**